src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -301,6 +299,31 @@ private void registerBuiltins() {
         reg(Material.HOPPER_MINECART, new ItemMinecart(GlowMinecart.MinecartType.HOPPER));
         reg(Material.POWERED_MINECART, new ItemMinecart(GlowMinecart.MinecartType.FURNACE));
         reg(Material.STORAGE_MINECART, new ItemMinecart(GlowMinecart.MinecartType.CHEST));
+        
+        //Wearable Items
+        reg(Material.LEATHER_BOOTS, new ItemType().wear(1, ItemWearablePosition.FEET, true));
```

Comment:
Equipping armor with right click was working before. Don't include a fix for it in this PR, since it's a regression somewhere else and adding a new implementation for this feature won't fix the root issue. Though, I appreciate pointing my attention to this new bug.